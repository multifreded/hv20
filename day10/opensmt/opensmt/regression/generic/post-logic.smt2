(set-logic AB_CD)
(declare-sort U 0)
(declare-fun c15 () U)
