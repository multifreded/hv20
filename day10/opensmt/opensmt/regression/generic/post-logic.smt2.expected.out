(error "unknown logic AB_CD")

(error "illegal command before set-logic: declare-sort")

(error "Illegal command before set-logic: declare-fun")

